### PR TITLE
Initial English translation for all of the documentation

### PR DESCRIPTION
**Pull request by ryanbnl**

### CODE REVIEW

File: README.EN.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+The intellectual property rights to (part of) this work rest with the Kingdom of the Netherlands. The Kingdom of the Netherlands reserves the right to this (part of the) work.
+
+The Kingdom of the Netherlands may use and publish the designs and documentation to which you contribute. By making a contribution, you transfer the intellectual property and copyright to the Kingdom of the Netherlands and agree that your contribution may also be used and made public by the Kingdom of the Netherlands.
+
+---
+
+NOTE: This README is also available [in English](README.EN.md). Thank you for understanding that the English version might not always be up-to-date with the Dutch one.
+
+NOTE: Deze README is ook beschikbaar [in het Engels](README.EN.md). Dank voor je begrip dat de Engelse versie niet altijd up-to-date is
```
Comment (JolandaVerhoef) [inline README.EN.md:65]: In the English Readme, I was expecting a link to the Dutch version. See https://github.com/corona-warn-app/cwa-documentation/blob/master/translations/README.de.md for an example of this.

File: README.EN.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+
```
Comment (JolandaVerhoef) [inline README.EN.md:1]: What was your consideration putting this Readme at the top-level directory? I think it makes sense to move this file to the translations directory as well. See https://github.com/corona-warn-app/cwa-documentation/blob/master/translations/README.de.md

File: translations/design-principles.en.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+### The app is designed for all audiences with a variety of demographic backgrounds and limitations (inclusivity).
```
Comment (JolandaVerhoef) [inline translations/design-principles.en.md:1]: Can you add the link to the Dutch (leading) version in this file as well?

File: translations/product-definition.en.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+## Product definition
```
Comment (JolandaVerhoef) [inline translations/product-definition.en.md:1]: Can you add the link to the Dutch (leading) version in this file as well?

File: translations/job-stories.en.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+# User Stories / Jobs to be Done
```
Comment (JolandaVerhoef) [inline translations/job-stories.en.md:1]: Can you add the link to the Dutch (leading) version in this file as well?

File: translations/usability-research.en.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+# Usability research
```
Comment (JolandaVerhoef) [inline translations/usability-research.en.md:1]: Can you add the link to the Dutch (leading) version in this file as well?

File: README.EN.md (hunk 2 above)
Comment (ryanbnl) [reply README.EN.md]: Oops we missed that one ;)

**Comment by ryanbnl** [comment]
@JolandaVerhoef I've added the backlinks, I thought it was overkill but given that we've also translated the filenames it's required.

**Comment by JolandaVerhoef** [comment]
Can you make sure that all commits are signed? See [this explanation](https://help.github.com/en/github/authenticating-to-github/signing-commits). This prevents spoofing of accounts.

**Comment by ryanbnl** [comment]
@JolandaVerhoef I have that setup now, do you want me to rebase -S or are the commits to-date ok? That'll sign Lennert's commits too.

**Comment by jelleprins** [comment]
I get a "Merging is blocked" - The base branch requires all commits to be signed.

Amazing work @ryanbnl & @lennert212 🔥

**Comment by JolandaVerhoef** [comment]
@ryanbnl all the commits need to be signed, otherwise we can't merge the PR. Can you rebase the branch?

**Comment by lennert212** [comment]
Working on getting a GPG key

**Comment by ryanbnl** [comment]
Hope that worked

**Comment by jelleprins** [comment]
Still getting the same 🛑 Merging is blocked.

**Comment by lennert212** [comment]
Still working on getting my GPG key...

**Comment by lennert212** [comment]
Ok. GPG keys associated with my Github  account.

**Comment by ryanbnl** [comment]
OK with some git-fu I have signed most of the commits; however I can't get `git rebase -i -p` to let me sign 28b12c0 and 4a17966. The -p is required to preserve previous merges.

I'm using the method described here:

https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits

Where you insert an `x git commit -S -s --amend --no-edit` after the commits you want to sign.

Does anyone have an idea?

At least this is the last time I'll skip working in a branch "because it's just documentation".

**Comment by ryanbnl** [comment]
git log --graph to the rescue. everything in this commit is signed now, with my sig but the author has been preserved.

**Pull request merged**